distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,120 @@
+priority "grant";
+// todo symlinks
+
+deny {
+    // Read, write, or execute any file
+    permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+    // Deny deletion of the security policy file through any Java code
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}default.policy", "delete";
```

Comment:
Should we block write too?
import will need write access though to restore that file.

Reply:
The permission line above this is already globally blocking read, write, and execute.

Reply:
We do grant many `codeBase`s access to `etc`, but it's _mostly_ restricted. We will lock it down more, but this should be sufficiently safe for now.